### PR TITLE
Update Bootstrap and theme variables

### PR DESCRIPTION
**Pull request by dennisreimann**
WIP until @dstrukt and I refine the sizing variables.

### CODE REVIEW

**Review by dstrukt — APPROVED** [review]
All looks good, already sent DMs about specific visual inconsistencies that have arrived, might post them here for good measure, but nice work!

File: BTCPayServer/wwwroot/main/site.css [hunk 1]
```diff
@@ -8,12 +8,6 @@ h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
     font-weight: 600;
 }
 
-/*.table,
-.table-sm {
-    width: calc(100% + 1rem);
-    margin: 1.5rem -.5rem;
-}*/
-
```
Comment (dennisreimann) [inline BTCPayServer/wwwroot/main/site.css]: This pulled the table out of the grid by the width of the horizontal padding. The goal was to align the table content with the rest of the content -- now this isn't the case anymore:

alignment

Also the vertical margins are missing without this. Was this an intended update?

**Comment by dennisreimann** [comment]
@dstrukt I addressed the points you sent via Mattermost. Can you confirm I did what you proposed? If so, let's get this one merged and iterate further in another PR.

File: BTCPayServer/wwwroot/main/site.css (hunk 1 above)
Comment (dstrukt) [reply BTCPayServer/wwwroot/main/site.css]: ahh, must have been removed when i was commenting, you can reverse (will comment this)

Comment (dennisreimann) [reply BTCPayServer/wwwroot/main/site.css]: Thanks, re-added in dac4de1

**Comment by dstrukt** [comment]
Can confirm, and added a few additional changes (for review) to address the issues my requested changes created!

**Comment by dennisreimann** [comment]
Let's get this merged, will rebase #2794 afterwards – that one is also basically done.

**Comment by pavlenex** [comment]
It's a great update but some things are a bit weird. Like we've started to bold text and use way too much light green on the warnings box that it hurts my eyes.

Screenshot 2021-09-01 at 14 19 52
Screenshot 2021-09-01 at 14 20 13


Screenshot 2021-09-01 at 14 17 31
Screenshot 2021-09-01 at 14 17 44

Also, what's the reasoning behind expanding menu size?

Screenshot 2021-09-01 at 14 23 24
Screenshot 2021-09-01 at 14 23 37


Personally I prefer light text, light menus and I am not really sure what kind of UX benefits we get with these changes?

I think this PR overdid certain things to the extand it made them look worse than before.

**Comment by dennisreimann** [comment]
@pavlenex As discussed on Mattermost we reverted the sidebar font size change and also reset the font weight for links back to normal instead of semibold.

What else needs to be addressed so that we can move forward and get this merged?

**Comment by pavlenex** [comment]
@dennisreimann This needs to be compared side by side with what we already have. I'll do that in a bit. 
Also there's a conflict that needs to be resolved.

**Comment by dennisreimann** [comment]
Reduced the navbar padding for desktop view now (Reverted to how it was). The sidebar font being 16px might be a caching issue -- can you check out the latest commit, rebuild and then hard refresh the browser? (Shift Reload or clear the browser cache)

**Review by pavlenex — APPROVED** [review]
Had to re-clone the repo for some reason, but it works now. All good, thanks reverting.

tACK

**Pull request merged**